server/routes/createGoal/createGoalsController.ts

```diff
@@ -35,8 +37,17 @@ export default class CreateGoalsController {
   // TODO: RR-748 - Implement submit handler for new create goal journey
   submitCreateGoalsForm: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber } = req.params
-    req.session.createGoalsForm = { ...req.body }
-    const { createGoalsForm } = req.session
+
+    const createGoalsForm = { ...req.body } as CreateGoalsForm
```

Comment:
We have to do this type of thing in the induction controllers as well - basically when we parse the request body into the form object, it may not have its arrays initialised.
IE. if the form is submitted with missing/blank step titles, given that the step title is the only field in the steps array, we end up with a goal with no steps array at all (rather than an empty array)
Equally, if the entire form were submitted having filled out not fields at all, then we end up with no goals array at all (rather than an empty array)
In both of these cases, where the array property is null/undefined rather than empty, the validator fails horribly!